conda-requirements

```diff
@@ -11,4 +11,3 @@ numexpr==2.4.4
 Cython==0.21
 networkx==1.10
 pytest==2.6.3
-pytest-html==1.8.1
```

Comment:
for now please add the correct version to conda-requirements. We can discuss removing this file entirely in a new issue/PR.